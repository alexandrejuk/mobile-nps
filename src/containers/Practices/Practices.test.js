import React from 'react'
import { render } from '@testing-library/react'

import PracticesContainer from './'

const goodPractices = [
  {
    title: "Guia de boas práticas",
    subtitle: "Resumo dos Procedimentos para Auditoria TCP",
    message: "Material Básico: 5 Copos, 1 Par de luvas, Touca, Jarra, Pincel, Papel toalha, Balança e  Tabela de programação.",
    image: 'PT0',
  },
  {
    title: "Procedimento 1",
    message: "Colocar a touca ao entrar no Ponto.",
    image: 'PT1',
  },
]

describe('Practices container', () => {
  it('should render correctly', () => {
    const callback = jest.fn()
    const step = 0
    const { container, getByText } = render(
      <PracticesContainer
        close={callback}
        onPrevStep={callback}
        onStep={callback}
        step={step}
        steps={(goodPractices.length - 1)}
        practice={goodPractices[step]}
        progress={25}
      />
    )

    getByText('Start')
    expect(container).toMatchSnapshot()
  })
})
