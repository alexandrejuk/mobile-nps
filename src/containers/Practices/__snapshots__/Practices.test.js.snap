// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Practices container should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="closeTools"
      role="button"
    >
      <img
        alt="tool close"
        src="close.svg"
      />
    </div>
    <div
      class="padding20"
    >
      <h1
        class="primary medium center title bold"
      >
        Guia de boas práticas
      </h1>
      <h1
        class="secondary medium center title normal"
      >
        Resumo dos Procedimentos para Auditoria TCP
      </h1>
    </div>
    <div>
      <div
        class="image"
      >
        <img
          alt="good practices"
          src="pt0.png"
        />
      </div>
      <div
        class="padding20"
      >
        <h1
          class="secondary medium center title normal"
        >
          Material Básico: 5 Copos, 1 Par de luvas, Touca, Jarra, Pincel, Papel toalha, Balança e  Tabela de programação.
        </h1>
      </div>
    </div>
    <div
      class="stepButton"
    >
      <div
        class="firtButton"
      >
        <button
          class="button primary"
          type="button"
        >
          Start
        </button>
      </div>
    </div>
  </div>
</div>
`;
